docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+mTLS (mutual TLS) is a variant where both client and server authenticate each other by verifying the provided certificates.
+This ensures secure identification and access control for API calls made from the SDK clients.
+Configuring SSL or mTLS authentication provides network encryption and protects against man-in-the-middle attacks when communicating with the Temporal server.
+
+## Built-in handler list Query
+
+| Feature                                                     | Go | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in handler list query](#built-in-handler-list-query) | No | No   | No         | No     | No   | No  |
+
+The built-in handler list Query in Temporal allows retrieving the list of registered Signal and Query handlers for a Workflow.
+
+When this Query is sent to a Workflow, the response contains a JSON serialized list of the following information about each handler:
+
+- Handler type: either Signal or Query
+- Handler name
+- Workflow function that handles the Signal or Query
+
+This can be useful for debugging or introspecting which Signal or Query a Workflow currently supports.
+
+## Built-in Temporal CLI dev server runner
+
+| Feature                                                                             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in Temporal CLI dev server runner](#built-in-temporal-cli-dev-server-runner) | Yes | No   | Yes        | Yes    | Yes  | No  |
+
+This allows starting the CLI dev server from code rather than manually from the terminal.
+It handles starting and stopping the server automatically around your Workflows.
+
+## Bulk replayer
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Bulk replayer](#bulk-replayer) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+The bulk replayer enables efficient parallel replay of many Workflow histories from a JSON file or an array of Workflow history event objects for testing and debugging Workflows in Temporal.
+
+## Cancellation
+
+| Feature                       | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Cancellation](#cancellation) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Cancellation in Temporal SDKs refers to the ability to stop the execution of a Workflow, Activity, or other operation before it completes normally.
+
+Each SDK provides APIs and abstractions to enable robust cancellation handling in Workflows and Activities.
+
+## Child Workflows
```

Comment:
These kinds of things are table stakes. This is implied by saying "SDKs support workflows" in the initial part. If it is not implied, there are dozens of other table-stakes workflow features that you can list here, child workflows and continue as new and such are not deserving as a top-level section.